content/en/docs/tasks/security/authorization/authz-ingress/index.md

```diff
@@ -217,13 +217,13 @@ metadata:
   namespace: foo
 spec:
   configPatches:
-  - applyTo: LISTENER
+  - applyTo: LISTENER_FILTER
     patch:
-      operation: MERGE
+      operation: INSERT_FIRST
       value:
-        listener_filters:
-        - name: envoy.listener.proxy_protocol
-        - name: envoy.listener.tls_inspector
+        name: proxy_protocol
+        typed_config:
```

Comment:
should we update tests/integration/pilot/gw_topology_test.go to do this method as well

Reply:
yeah, it's done in https://github.com/istio/istio/pull/45223